frontend/components/Navbar/Narbar.tsx

```diff
@@ -0,0 +1,34 @@
+import Link from "next/link";
+import styles from "./navbar.module.css";
+import Head from "next/head";
+
+function Navbar() {
+  return (
+    <div className={styles.background}>
+      <div className={styles.logo}>
+        <h1 className={styles.title}>Portfolio Creator</h1>
+      </div>
+      <div className={styles.middle}>
+        <ul className={styles.navlist}>
+          <li>
+            <Link href="/">Home</Link>
+          </li>
+          <li>
+            <Link href="/about">About</Link>
+          </li>
+          <li>
+            <Link href="/portfolio">Templates</Link>
+          </li>
```

Comment:
As mentioned before, I think the `About` link should come after the `Templates` link.
```suggestion
          <li>
            <Link href="/portfolio">Templates</Link>
          </li>
          <li>
            <Link href="/about">About</Link>
          </li>
```